libraries/standard/mqtt/src/mqtt.c

```diff
@@ -25,7 +25,8 @@
 
 /*-----------------------------------------------------------*/
 
-static int32_t sendPacket( MQTTContext_t * pContext, size_t bytesToSend )
+static int32_t sendPacket( MQTTContext_t * pContext,
```

Comment:
_sendPacket() since you are changing this line anyway.

Reply:
Thanks. Will update.